core/core.go

```diff
@@ -27,23 +27,30 @@ func Import(env *vm.Env) *vm.Env {
 	})
 
 	env.Define("range", func(args ...int64) []int64 {
-		if len(args) < 1 {
-			panic("Missing arguments")
-		}
-		if len(args) > 2 {
-			panic("Too many arguments")
-		}
-		var min, max int64
-		if len(args) == 1 {
-			min = 0
-			max = args[0] - 1
-		} else {
-			min = args[0]
-			max = args[1]
+		var start, stop int64
+		var step int64 = 1
+
+		switch argsLen := len(args); argsLen {
+		case 0:
+			panic("range expected 1 arguments, got 0")
```

Comment:
Change to something like: range needs at least 1 argument, got 0

Reply:
remove it

Reply:
done